src/Symfony/Component/Yaml/Inline.php

```diff
@@ -562,6 +562,8 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                         }
 
                         return;
+                    case 0 === strpos($scalar, '!php/const:'):
+                        return @constant(substr($scalar, 11));
```

Comment:
It's not an error, it's a warning, preventing to get `null`, remove the `@` and the last test fail because it gets `0.0` instead of `null`. That's why I silenced it (and removed the flag in the process without giving it a second thought).